lib/px/block/pxtemplate.lua

```diff
@@ -111,6 +64,15 @@ function M.load(config_file)
         return content
     end
 
+    local function get_script(template, px_config)
```

Comment:
I suggest calling this function inside `get_props`
https://github.com/PerimeterX/perimeterx-nginx-plugin/blob/44bdd6197d5c7ac844a9d8ae609aaed5093e5e0d/lib/px/block/pxtemplate.lua#L78

and calling `get_props` on line #81
https://github.com/PerimeterX/perimeterx-nginx-plugin/blob/44bdd6197d5c7ac844a9d8ae609aaed5093e5e0d/lib/px/block/pxtemplate.lua#L81